addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -418,39 +426,66 @@
+         t-att-data-question-id="question['id']" t-att-data-title="question['question']" >
+        <div class="row d-flex mb-2 mx-0">
             <div class="h4">
-                <small class="text-muted"><span t-esc="question_index+1"/>. </small> <span t-esc="question['question']"/>
+                <small class="text-muted">
+                    <i class="o_wslides_js_quiz_sequence_handler fa fa-bars mr-1" t-if="slide.channel_id.can_upload and not slide_completed" />
+                    <t t-if="question_index != NoneType"><span class="o_wslides_quiz_question_sequence" t-esc="question_index+1"/>.</t>
+                    <t t-else=""><span class="o_wslides_quiz_question_sequence" t-esc="question['sequence']"/>.</t>
+                </small>
+                <span t-esc="question['question']"/>
             </div>
-            <div class="list-group">
-                <t t-foreach="question['answers']" t-as="answer">
-                    <a t-att-data-answer-id="answer['id']" href="#"
-                        t-att-data-text="answer['text_value']"
-                        t-att-class="'o_wslides_quiz_answer list-group-item list-group-item-action d-flex align-items-center %s' % ('list-group-item-success' if slide_completed and answer['is_correct'] else '')">
-                        <label class="my-0 d-flex align-items-center justify-content-center mr-2">
-                            <input type="radio"
-                                t-att-name="question['id']"
-                                t-att-value="answer['id']"
-                                class="d-none"
-                                t-att-disabled="True if not slide.channel_id.is_member or slide_completed else ''"/>
-                            <i t-att-class="'fa fa-circle text-400 %s' % ('d-none' if slide_completed and answer['is_correct'] else '')"/>
-                            <i class="fa fa-times-circle text-danger d-none"></i>
-                            <i t-att-class="'fa fa-check-circle text-success %s' % ('d-none' if not (slide_completed and answer['is_correct']) else '')"></i>
-                        </label>
-                        <span t-esc="answer['text_value']"/>
-                    </a>
-                </t>
+            <div class="ml-auto o_wslides_js_quiz_edit_del" t-if="slide.channel_id.can_upload and not slide_completed" >
+                <i class="o_wslides_js_quiz_edit_question fa fa-pencil-square-o p-1"></i>
+                <i class="o_wslides_js_quiz_delete_question fa fa-trash p-1"></i>
             </div>
         </div>
-        <div class="alert alert-danger o_wslides_js_lesson_quiz_error mt-2 d-none" role="alert">
-            <span></span>
-            <button type="button" class="close" data-dismiss="alert" aria-label="Close">
-                <span aria-hidden="true">&amp;times;</span>
-            </button>
+        <div class="list-group">
+            <t t-foreach="question['answer_ids']" t-as="answer">
```

Comment:
This seems to be regular python evaluation.
Can't you do `question.answer_ids` instead of `question['answer_ids']` ?

Same for all other ['prop'] accesses.

Reply:
I tried but it doesn't work.
In fact the returned object from the _get_slide_quiz_data method is a dict